modules/web/site/main.bicep

```diff
@@ -269,6 +277,7 @@ resource app 'Microsoft.Web/sites@2021-03-01' = {
     hostNameSslStates: hostNameSslStates
     hyperV: hyperV
     redundancyMode: redundancyMode
+    publicNetworkAccess: !empty(publicNetworkAccess) ? any(publicNetworkAccess) : (!empty(privateEndpoints) ? 'Disabled' : 'Enabled')
```

Comment:
@chris5287 The property seems to be something added after the `2021-03-01` API.

![image](https://github.com/Azure/ResourceModules/assets/28486158/81e9694b-0247-4979-be13-41343c8df531)

![image](https://github.com/Azure/ResourceModules/assets/28486158/8a8d7d58-fff3-4c1d-9539-0fbbf8f7736b)
https://learn.microsoft.com/en-us/azure/templates/microsoft.web/change-log/sites

Can you please bump up the API version of the module to support this change? Please let me know if you do not have a test environment to validate this change. Thank you

Reply:
I've bumped, not sure if I should add the other new properties as well?

![image](https://github.com/Azure/ResourceModules/assets/1552744/aeee5e66-6f76-4220-a721-ed39065c1be9)